### PR TITLE
Loader indicator

### PR DESCRIPTION
**Pull request by jamesnakagawa**
Add a loader indicator to all search interfaces.

### CODE REVIEW

**Comment by jamesnakagawa** [comment]
Also contains a small display adjustment to buttons.

File: kive/container/templates/container/batch_form.html [hunk 1]
```diff
@@ -68,6 +68,16 @@ <h3>Container Runs:</h3>
...
+                <div class="la-ball-spin la-sm">
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                </div>
```
Comment (donkirkby) [inline kive/container/templates/container/batch_form.html]: The effect is beautiful, but I'm a little concerned about how much boilerplate is needed in each page, twice. Could the `PermissionsTable` class add all this, possibly into a div with an expected class?

Comment (jamesnakagawa) [reply kive/container/templates/container/batch_form.html]: Agreed, it's a bit much. It's a little faster this way (dynamically inserting elements is expensive), but probably wouldn't result in a noticeable slowdown. But is there something we could do with Django templates instead? The markup for these all look pretty similar.

Comment (donkirkby) [reply kive/container/templates/container/batch_form.html]: It looks like the Django [include] tag would be useful here.

[include]: https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#include

Comment (donkirkby) [reply kive/container/templates/container/batch_form.html]: Of course, the whole table is being built by dynamically inserting elements every time you refresh. Is dynamically inserting a few divs at page load even 1% of that? I suggest you look at the include tag, and take the performance win if it's really easy. If it's at all annoying, just go with the Javascript option.

Comment (donkirkby) [reply kive/container/templates/container/batch_form.html]: I moved the basic search field and the advanced submit button into templates, and it works well. I'll merge this branch.

**Pull request merged**